src/main/java/io/pravega/perf/PerfStats.java

```diff
@@ -279,7 +418,7 @@ public void printTotal(long endTime) {
             int[] percs = getPercentiles();
 
             log.info(String.format(
-                    "%d records %s, %.3f records/sec, %d bytes record size, %.2f MiB/sec, %.1f ms avg latency, %.1f ms max latency" +
+                    "%,d records %s, %,.3f records/sec, %d bytes record size, %.2f MiB/sec, %.1f ms avg latency, %.1f ms max latency" +
```

Comment:
Just to note that I have just remembered that I added commas in several formats to make the numbers easier to read, i.e. `2,684,355 records Writing`.  I'm not sure if anybody is scraping these numbers directly from the output log which would get confused by the commas.

Reply:
I like the commas for readability. Check with Tim Butler to see if he scrapes these in his scripts.